views/openweathermap-form-settings.twig

```diff
@@ -58,5 +58,9 @@
         {% set title %}{% trans "Cache Period" %}{% endset %}
         {% set helpText %}{% trans "This module uses 3rd party data. Please enter the number of seconds you would like to cache results." %}{% endset %}
         {{ forms.input("cachePeriod", title, interface.getSetting("cachePeriod", 3600), helpText) }}
+
+        {% set title %}{% trans "XMDS Cache Period" %}{% endset %}
```

Comment:
The user might not know what XMDS means, so I think we can set this title to "Schedule Criteria Cache Period"